pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -28,6 +28,8 @@ import (
 	"go.goms.io/fleet/pkg/utils/controller"
 )
 
+const CustomResourceDefinition = "CustomResourceDefinition"
```

Comment:
no need to create the const, you can use the string directly as it's used in one place only.

Reply:
I had to create because of a lint failure. It suggested to do that.